lib/hexpm/accounts/session.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Hexpm.Accounts.Session do
```

Comment:
But in this case the schema is actually part of the web portion of my application. I'm not sure what the workaround is though? Hide it behind another behaviour or maybe use schemaless with just an ecto source?